cmake/BLTMacros.cmake

```diff
@@ -663,3 +663,56 @@ macro(blt_append_custom_compiler_flag)
 
 endmacro(blt_append_custom_compiler_flag)
 
+
+
+
+##------------------------------------------------------------------------------
+## blt_find_libraries( <FOUND_LIBS>
+##                     NAMES [libname1 [libname2 ...]]
+##                     PATHS [path1 [path2 ...]] )
+##
+## This command is used to find a list of libraries. A cache entry named by <FOUND_LIBS> 
+## is created to store the result of this command. If the libraries are found the
+## results are stored in FOUND_LIBS.
+##
+## NAMES lists the names of the libraries to search for.
+##
+## PATH lists the paths in which to search for NAMES.
+##
+##
+##------------------------------------------------------------------------------
+macro(blt_find_libraries FOUND_LIBS)
```

Comment:
Could you please add an example of the usage of this to the blt tests?
I suppose it should be for a system library that should be available on all systems (or guarded if it is only for unix-based systems).